File: PHP/1094.php
<?php

$n = (int) readline();
$c = 0;
$r = 0;
$s = 0;

for ($i = 0; $i<$n; $i++){
	$resp = explode(' ', readline());
	if ($resp[1] == 'C')
		$c += (int) $resp[0];
	else if ($resp[1] == 'R')
		$r += (int) $resp[0];
	else
		$s += (int) $resp[0];
}

$total = $c + $r + $s;
$pc = number_format($c * 100 / $total, 2);
$pr = number_format($r * 100 / $total, 2);
$ps = number_format($s * 100 / $total, 2);
echo <<<EOF
Total: {$total} cobaias
Total de coelhos: {$c}
Total de ratos: {$r}
Total de sapos: {$s}
Percentual de coelhos: {$pc} %
Percentual de ratos: {$pr} %
Percentual de sapos: {$ps} %\n
EOF;

?>

File: Python/1094.py
if __name__ == '__main__':
    n = int(input())
    coelhos = 0
    ratos = 0
    sapos = 0
    for i in range(0, n):
        resp = input().split(' ')
        if resp[1] == 'C':
            coelhos += int(resp[0])
        elif resp[1] == 'R':
            ratos += int(resp[0])
        else:
            sapos += int(resp[0])
    cobaias = coelhos + ratos + sapos 
    print(f'Total: {cobaias} cobaias')
    print(f'Total de coelhos: {coelhos}')
    print(f'Total de ratos: {ratos}')
    print(f'Total de sapos: {sapos}')
    print(f'Percentual de coelhos: {coelhos*100/cobaias:.2f} %')
    print(f'Percentual de ratos: {ratos*100/cobaias:.2f} %')
    print(f'Percentual de sapos: {sapos*100/cobaias:.2f} %')

